web/pageComponents/shared/siteMenu/simple/SimpleSiteMenu.tsx

```diff
@@ -55,6 +54,13 @@ const SimpleSiteMenu = ({ data, ...rest }: MenuProps) => {
     setIsOpen(false)
   }, [])
 
+  const { refs, context } = useFloating({
+    open: isOpen,
+    onOpenChange: setIsOpen,
+  })
+
+  const { getReferenceProps, getFloatingProps } = useInteractions([useDismiss(context)])
+
```

Comment:
Maybe move this part into its own component as it is used twice

Reply:
its two hooks and i don't think  you can't wrap it in a separate component as you need to set refs floating and reference on the return elements  and you spread the Props on the respective return elemets